lms/templates/fields/field_readonly.underscore

```diff
@@ -1,10 +1,8 @@
-<label class="u-field-title" aria-hidden="true">
-    <%- title %>
-</label>
-<span class="sr" for="u-field-input-<%- id %>"><%- screenReaderTitle %></span>
-<span class="u-field-value">
-    <input id="u-field-input-<%- id %>" aria-describedby="u-field-message-<%- id %>" type="text" name="input" readonly=true value="<%- value %>">
-</span>
+<% if (title) { %>
+<span class="u-field-title" aria-hidden="true"><%- title %></span>
```

Comment:
@cahrens @cptvitamin  I'm still curious what the cases are where the screenReaderTitle would differ from title, and whether the distinction is really needed.  I'm mainly wondering because `.u-field-value` element already has an `aria-describedby` attribute, so a separate title seems excessive to me.  Does the description not do what I think it does?